docs/index.md

```diff
@@ -180,6 +180,15 @@ ViewComponent is built by over a hundred members of the community, including:
 <img src="https://avatars.githubusercontent.com/mattwr18?s=64" alt="mattwr18" width="32" />
 <img src="https://avatars.githubusercontent.com/dylanatsmith?s=64" alt="dylanatsmith" width="32" />
 
+## Who uses ViewComponent?
+
+* [GitHub](https://github.com/) (900+ components used 15k+ times)
```

Comment:
Out of curiosity, @joelhawksley, how is the team pulling stats?

Reply:
You can run `rails stats` to get stats about various parts of the app. ViewComponents are listed there since #1050.

But I'm curious too about the "used 15k+ times" figure :)

Reply:
I had forgotten about that! @Spone would you be up for writing an addition to the docs?

@dylanatsmith we just search for `.new` in our `.erb` files 😄

Reply:
I'd be happy to, but I'm not sure where, now that the FAQ is gone... :laughing: Instrumentation? Compatibility?

Reply:
Maybe this could be a new section in the Guide, since it is a feature?